common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -206,6 +206,16 @@ trait FeatureSwitches {
     exposeClientSide = false
   )
 
+  val IdentityGdprV2NewslettersConsents = Switch(
+    SwitchGroup.Identity,
+    "id-gdpr-use-v2-newsletters-list-ids-in-edit-profile",
+    "If switched on, users will subscribe to new V2 newsletter lists in email-prefs and consents journey",
+    owners = Seq(Owner.withGithub("mario-galic")),
+    safeState = Off,
+    sellByDate = new LocalDate(2018, 1, 15),
```

Comment:
I wonder whether we should give these switches all a few more days leeway so we're not having to tidy them all up on the 15th.

Reply:
I'm working on a PR about that, let's take this talk there ^^

Reply:
Yes good idea. Separate PR.